src/mscorlib/model.xml

```diff
@@ -3434,6 +3434,7 @@
       <Member Status="ImplRoot" Name="#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)" />
       <Member Name="Add(System.IntPtr,System.Int32)" />
       <Member Name="Equals(System.Object)" />
+      <Member Status="ImplRoot" Name="System.IEquatable&lt;System.IntPtr&gt;.Equals(System.IntPtr)" />
```

Comment:
Do you really these as ImplRoot?

(In theory, these should not be marked as ImplRoot at all because of the interface implementation keeps them alive.)